express-spotify-app/app.js

```diff
@@ -0,0 +1,67 @@
+const express = require('express');
+const app = express();
+const hbs = require('hbs');
+
+var SpotifyWebApi = require('spotify-web-api-node');
+
+app.use(express.static(__dirname + "/public"));
+
+app.set ("views", __dirname + "/views");
+app.set("view engine", "hbs")
+app.set ("layout", __dirname + "/views/layout.hbs");
+
+// Routes 
+// ------------
+app.get ("/", (req, res, next)=>{
+  res.render("home-random");
+});
+
+app.get ("/artists", (req, res, next)=>{
+  //actions 
+  spotifyApi.searchArtists(req.query.searchterm)
+  .then(data => {
+    console.log(data)
+    res.render("artists");
```

Comment:
You are not sending to the view anything from the `data` you fetched from the spotifyAPI, so you won't be able to show anything to the user. Use `res.locals` or a second argument to `res.render`